apps/api/src/evm/writers.ts

```diff
@@ -641,7 +646,7 @@ export function createWriters(config: FullConfig) {
     );
 
     if (executionStrategy) {
-      proposal.quorum = BigInt(executionStrategy.quorum);
+      proposal.quorum = executionStrategy.quorum.toString();
```

Comment:
This is not great solution I think, as mentioned [there](https://github.com/snapshot-labs/workflow/issues/581#issuecomment-2983290811).

Issue would be if we do this.
1. Quorum is 1.
2. I create proposal.
3. Proposal is still open I change quorum to 1000.
4. Proposal get votes and crosses old quorum (but not the new one).
5. UI will think we can execute it but we can't.